src/JustEat.StatsD/JustEat.StatsD.csproj

```diff
@@ -1,11 +1,17 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <Description>A .NET library for publishing metrics to statsd.</Description>
     <OutputType>Library</OutputType>
     <PackageId>JustEat.StatsD</PackageId>
     <RootNamespace>JustEat.StatsD</RootNamespace>
     <Summary>A .NET library for publishing metrics to statsd.</Summary>
-    <TargetFrameworks>net451;netstandard2.0</TargetFrameworks>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
```

Comment:
If we're a generic library, we shouldn't be targeting `netcoreappx.x`

Reply:
This is the nature of the situation that has come about.
There is lots of cool perf stuff currently exclusive to .NET Core 2.1, and not other runtimes.
For example, today I think `Span<T>` wont be shipping with .NET 4.8 (see comments here: https://blogs.msdn.microsoft.com/dotnet/2018/06/06/announcing-net-framework-4-8-early-access-build-3621/).
We'll probably need to wait for .NET Standard 3.0, which could be a very long way off.

IMO, for this PR to be more acceptable, this needs to be an additional target, with no breaking changes or additional deps for existing targets.

Reply:
But then how long would we support that for in future versions? From what I've seen in conversations online from the .NET guys like Immo and David Fowler, the guidance seems to be that if you're a wide-ranging general library, you should wait for other implementations to catch-up (e.g. Mono) and for the standard to be updated, rather than rush into supporting a specific implementation to get access to the new shiny things ASAP.

Reply:
I've heard it more like "if you want the stuff we shipped in .NET Core 2.1, additionally target `netcoreapp2.1`, or wait until it hits the standard".

If done in a way that isn't detrimental to current consumers of this package, I think this PR is a great feature, and something to boast about.